core/domain/stats_domain.py

```diff
@@ -420,7 +420,7 @@ def __init__(self, exp_id, unresolved_issues):
                 represents an issue along with the associated playthroughs.
                 Each dict will be of the form:
                 {
-                    issue_type: str. Type of the issue.
+                    issue_id: str. ID of the issue.
```

Comment:
As an FYI I think issue_type is better. An ID uniquely identifies an issue.

(We got this wrong with interactions -- we should actually have used interaction_type instead of interaction_id, oh well. But let's not propagate that.)